x-pack/plugins/security_solution/server/lib/detection_engine/rules/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const EPR_PACKAGE = 'security_detection_engine';
```

Comment:
Dead variable? I don't see this used anywhere.

Reply:
Removed in 84248b3b98b6bb80d623152bf1c5b9eee6f22c5e

Reply:
i originally thought that we'd autoinstall the fleet package in this PR, then i descoped that. so i can add this back later if we get to that